### ISSUE TITLE
Re: 'Bank Holidays'

### ISSUE THREAD

**Issue by LordScree** [co-cddo/api-catalogue#300]
Regarding the 'Bank Holidays' entry (https://www.api.gov.uk/gds/bank-holidays/), the documentation link here points to a GitHub repository that was deprecated in May 2020.

Has the 'Bank Holidays' API itself been decommissioned? Or is this just an issue with the documentation link?

**Comment by dch82** [comment]
I don't think so, as the bank holidays go up to the end of 2027. A quick check on the repo shows that it only went up to 2021 when it was abandoned. I'm going to assume that someone updates it every now and then. (Also I realised that there is a dedicated column for bunting lol)
